### DISCUSSION TITLE
Run GUI from command line

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [samschott/maestral#724] [Q&A] [answered]
I did not understand how to run the GUI from command line.

If I run `maestral_qt`, it is run but the command does not return. The terminal stay open on that process.

Also do I need to run the daemon separately?

**Comment by samschott** [comment, marked as answer]
> If I run maestral_qt, it is run but the command does not return. The terminal stay open on that process.

That is working as intended. To run the GUI detached from a Terminal, you can create a desktop launcher (the process will vary depending on your desktop environment) or detach the GUI process with `maestral_qt&`.

> Also do I need to run the daemon separately?

The GUI will start a daemon on demand if it does not find a running sync daemon one.

**Reply by raffaelemancuso** [reply]
Thanks Sam!
